src/components/Documentation/SharedStyles.js

```diff
@@ -19,7 +19,7 @@ export const SideNavColumn = styled(Column)`
 export const DividerEl = styled.span`
   display: inline-block;
   width: 0.0625rem;
-  height: 1.5rem;
+  height: 2.75rem;
```

Comment:
This `DividerEl` is also being used in `<CodeExample/>` which has a shorter height (previous 1.5rem) value.

Reply:
Ah good callout, ty